packages/utils/src/validation/ObjectValidator.js

```diff
@@ -68,7 +68,7 @@ async function runValidators(validators, object, fieldKey, constructError) {
       await validators[j](object[fieldKey], object, fieldKey);
     } catch (error) {
       throw constructError
-        ? constructError(error.message, fieldKey)
+        ? constructError(error.message, fieldKey, { ...error?.extraFields })
```

Comment:
Spend a bit more time to figure out how it works here. Using `constructError` only for `constructImportValidationError` is confusing. It could be nicer to catch the error along with where it happens...